packages/icons/icons/drag-handle.svg

```diff
@@ -0,0 +1,6 @@
+<svg width="16" height="16" viewBox="0 0 16 16" fill="none" xmlns="http://www.w3.org/2000/svg">
+  <rect y="3" x="5" width="6" height="1" rx="0.5" />
+  <rect y="6" x="5" width="6" height="1" rx="0.5" />
+  <rect y="9" x="5" width="6" height="1" rx="0.5" />
+  <rect y="12" x="5" width="6" height="1" rx="0.5" />
+</svg>
```

Comment:
Not sure we should put drag handle in icons. Because it's not an icon from Figma point of view. I would probably inline svg in the component code.

And if it's an 16x16 icon you're probably have trouble with layout later as it should be smaller

<img width="861" alt="Screenshot 2023-02-27 at 19 31 00" src="https://user-images.githubusercontent.com/825702/221606986-d98d9279-8bdc-44b1-b9c0-ec6481c7354f.png">